### PR TITLE
Upload repos to tmp folder for Hardened images

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]

**Review by m-dati — APPROVED** [review]

**Comment by asmorodskyi** [comment]
will wait for VR to pass https://openqa.suse.de/tests/13280159

**Review by m-dati — COMMENTED** [review]
If still needed this, I suggest to use ls $repodir/..

File: tests/publiccloud/transfer_repos.pm [hunk 1]
```diff
@@ -53,7 +55,8 @@ sub run {
             assert_script_run("echo $repo | tee -a /tmp/transfer_repos.txt");
         }
         # VM repos.dir support preparation
-        $args->{my_instance}->ssh_assert_script_run("sudo mkdir $repodir;sudo chmod a+w $repodir");
+        $args->{my_instance}->ssh_assert_script_run("sudo mkdir $repodir;sudo chmod 777 $repodir");
+        $args->{my_instance}->ssh_assert_script_run("sudo ls -la /tmp");
```
Comment (m-dati) [inline tests/publiccloud/transfer_repos.pm:59]: If still needed this, I suggest to use `ls $repodir/..`

Comment (asmorodskyi) [reply tests/publiccloud/transfer_repos.pm:59]: but anyway I will remove that it was for testing purposes

**Pull request merged**